src/PartielsPy/Partiels.py

```diff
@@ -0,0 +1,170 @@
+"""A main class for Partiels Wrapper"""
+
+from pathlib import Path
+
+# from lxml import etree
+import os
+import shutil
+import json
+import subprocess
+from .Version import Version
+from .Document import Document
+from .Exporters.ImageExporter import ImageExporter
+from .Exporters.TextExporter import (
+    CsvExporter,
+    JsonExporter,
+    CueExporter,
+    ReaperExporter,
+    LabExporter,
+)
+from .Exporters.SdifExporter import SdifExporter
+
+PARTIELS_HANDLED_VERSION_MIN = "2.0.9"
+PARTIELS_HANDLED_VERSION_MAX = "2.0.10"
+
+
+class Partiels:
+
+    def __init__(self):
+        self.setExecPath(self.findExecPath())
+        self.isHandledVersion = self.checkVersion()
+
+    def getExecPath(self):
+        """Return the executable path of Partiels"""
+        return self.exec_path
+
+    def setExecPath(self, path: str):
+        """Set the path for Partiels's executable
+
+        Args:
+            path (str): relative or absolute path
+        """
+        self.exec_path = path
+
+    def setVampPath(self, path: str):
+        os.environ["VAMP_PATH"] = path
+
+    def findExecPath(self):
+        "Find Partiels's executable path"
+        exec_name = "Partiels"
+        exec_path = shutil.which(exec_name)
```

Comment:
The default locations should be tried, such as /Application/Partiels.app/Contents/MacOS or ‘ProgramFiles’. The user should also be told to use the PATH environment variable to locate a specific version.